Kontent.Ai.Delivery.Caching.Tests/Fixtures/full_articles.json

```diff
@@ -144,7 +144,7 @@
         "metadata__twitter_creator": {
           "type": "text",
           "name": "twitter:creator",
-          "value": "@kenticoCloud"
+          "value": "@kontentAi"
```

Comment:
not sure, but rather ask not @kontent-ai?

Reply:
I guess if we want it by the book, it should be @Kontent_ai :) https://twitter.com/Kontent_ai will update